### PR TITLE
check all pages can be loaded normally

### PR DESCRIPTION
**Pull request by gouyang**
To test that all pages are loaded normally:
1. revert overview test to ensure overview page is loaded
2. add new tests about VM/VMI tabs and template's tabs
3. the catalog tests should already cover the catalog page

### CODE REVIEW

**Comment by gouyang** [comment]
now it see two `Virtualization` in the nav.

**Comment by avivtur** [comment]
@gouyang Hi :) 
can you please try and rebase your PR and maybe it will fix the failing tests?

**Comment by gouyang** [comment]
@avivtur I've rebased the PR and the tests got the same results.

**Comment by avivtur** [comment]
@gouyang  can you reproduce this issue locally in any way?

**Comment by upalatucci** [comment]
No, we can't reproduce it locally.

File: cypress/tests/check-tabs.spec.ts [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+      cy.visitVMs();
+      vm.createVMFromYAML();
+      cy.contains('Hostname').should('be.visible');
+
+      // start the vm
+      cy.byButtonText('Actions').click();
+      cy.get(vmView.vmActionStart).click();
+      cy.contains(vmView.vmStatusOnOverview, 'Starting').should('exist');
+      // VM in CI cannot become running
+      // cy.contains(vmView.vmStatusOnOverview, 'Running', { timeout: 600000 }).should('exist');
```
Comment (avivtur) [inline cypress/tests/check-tabs.spec.ts:29]: is this left intentionally?

File: cypress/tests/check-tabs.spec.ts [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+    });
+
+    it('vm tabs are loaded', () => {
```
Comment (avivtur) [inline cypress/tests/check-tabs.spec.ts:32]: I think it should also make sure that the overview page is also loaded before moving to details tab, wdyt?

File: cypress/support/nav.ts [hunk 3]
```diff
@@ -1,19 +1,43 @@
...
+  cy.contains(resourceTitle, 'VirtualMachines').should('be.visible');
+  // eslint-disable-next-line cypress/no-unnecessary-waiting
+  cy.wait(1000);
+});
+
+Cypress.Commands.add('visitTemplates', () => {
+  cy.clickVirtLink(templateNav);
+  cy.contains(resourceTitle, 'VirtualMachine Templates').should('be.visible');
+  // eslint-disable-next-line cypress/no-unnecessary-waiting
+  cy.wait(1000);
```
Comment (avivtur) [inline cypress/support/nav.ts:42]: do we also want to visit DataSources and MigrationPolicies page?

**Review by avivtur — COMMENTED** [review]
@gouyang great job!
I left some comments but it can also be fixed in a follow-up PR

File: cypress/tests/check-tabs.spec.ts (hunk 1 above)
Comment (gouyang) [reply cypress/tests/check-tabs.spec.ts:29]: yes, it seems the VM cannot get into running in the CI.
I added a comment in line 28.

File: cypress/tests/check-tabs.spec.ts (hunk 2 above)
Comment (gouyang) [reply cypress/tests/check-tabs.spec.ts:32]: There is a check about the overview tab in previous `it` block, though it's not explicit.

File: cypress/support/nav.ts (hunk 3 above)
Comment (gouyang) [reply cypress/support/nav.ts:42]: Will add a new PR about the new pages.

**Pull request merged**